lib/registries/addon/components/registries-registration-type-facet/component.ts

```diff
@@ -43,11 +36,14 @@ export default class RegistriesRegistrationTypeFacet extends Component {
     @waitFor
     async fetchRegistrationTypes() {
         try {
-            const metaschemas = await this.store.query('registration-schema', {
+            if (!this.provider){
+                this.provider = await this.store.findRecord('registration-provider', 'osf');
+            }
+            const metaschemas = await this.provider.queryHasMany('schemas', {
                 'page[size]': 100,
             });
             const metaschemaNames = metaschemas.mapBy('name');
-            if (!this.features.isEnabled(enableInactiveSchemas)) {
+            if (this.provider.id === 'osf') {
```

Comment:
This feature flag was valid for the top-level schemas endpoint, but is not valid for the provider-schemas. Since ERPC should only be visible if we're filtering of OSF Registries, I modified the conditional accordingly.